src/screens/Home/components/ChainListItem.js

```diff
@@ -68,23 +63,28 @@ const Container = styled.TouchableOpacity`
   align-items: center;
 `;
 
+const DeployContainer = styled.View`
+  flex: 1;
+  flex-direction: row;
+  align-items: center;
+`;
+
+const HazardIcon = styled(Icon)`
+  margin-left: 8px;
+`;
+
 const Title = styled(Text)`
   flex: 1;
   margin: ${spacing.mediumLarge}px 0 ${spacing.mediumLarge}px 36px;
   ${fontStyles.medium};
 `;
 
-const Value = styled(Text)`
+const TitleWithIcon = styled(Text)`
```

Comment:
The name is a bit confusing, as there is no icon in this particular component.

Reply:
Seem like if you removed `flex: 1` from Title you could share the same style with that.

Reply:
Done.